### PR TITLE
[INJICERT-421] w.r.t. esignet version bump to 1.4.1

### PR DESCRIPTION
**Pull request by Piyush7034**

### CODE REVIEW

File: artifacts/pom.xml [hunk 1]
```diff
@@ -208,14 +208,24 @@
...
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mock-plugin.version>0.2.1-SNAPSHOT</certify-mock-plugin.version>
+		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
 		
 		<!-- esignet IDA wrapper -->
 		<esignet-ida-wrapper.version>1.2.1-SNAPSHOT</esignet-ida-wrapper.version>
 		<esignet-ida-wrapper.fileName>esignet-ida-wrapper.jar</esignet-ida-wrapper.fileName>
-		<esignet-digital-credential-wrapper.version>0.1.0-SNAPSHOT</esignet-digital-credential-wrapper.version>
-                <esignet-digital-credential-wrapper.fileName>sunbird-rc-esignet-integration-impl.jar</esignet-digital-credential-wrapper.fileName>
+		<esignet-digital-credential-wrapper.version>0.2.0-SNAPSHOT</esignet-digital-credential-wrapper.version>
```
Comment (jainhitesh9998) [inline artifacts/pom.xml:227]: can you change this also to 0.2.1-snapshot, since its part of digital credentials plugin module

**Review by jainhitesh9998 — CHANGES_REQUESTED** [review]

File: artifacts/pom.xml (hunk 1 above)
Comment (Piyush7034) [reply artifacts/pom.xml]: Changed version to 0.2.1-snapshot for esignet-wrapper

File: artifacts/pom.xml [hunk 2]
```diff
@@ -208,14 +208,24 @@
...
+
+		<!-- certify mosip identity plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mosip-identity-plugin.version>0.2.1-SNAPSHOT</certify-mosip-identity-plugin.version>
+		<certify-mosip-identity-plugin.fileName>certify-mosip-identity-plugin.jar</certify-mosip-identity-plugin.fileName>
+
+		<!-- certify mock plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mock-plugin.version>0.2.1-SNAPSHOT</certify-mock-plugin.version>
+		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
```
Comment (vishwa-vyom) [inline artifacts/pom.xml:222]: I see the relevant artifactitem sections are missing

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: artifacts/pom.xml (hunk 2 above)
Comment (Piyush7034) [reply artifacts/pom.xml:222]: Added artifact items for certify-mock and mosip-identity plugins

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**